frontend/src/components/Order/components/DeliveryForm/DeliveryForm.jsx

```diff
@@ -0,0 +1,165 @@
+import React, { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import styles from './DeliveryForm.module.scss';
+import { ICONS } from '../../../ui-kit/icons';
+import Button from '../../../ui-kit/components/Button';
+
+const DeliverForm = ({ deliveryData, setDataOnSubmit }) => {
+  const [showShopList, setShowShopList] = useState(
+    deliveryData?.delivery
+      ? deliveryData.delivery == 'Pick up from the shop'
```

Comment:
Are you sure about this statement? It looks too complicated. Do we need that?

Reply:
Yes, we need it. We have 2 radio buttons. When the customer selects "Pick up from the shop", I need to show a list of shops. Otherwise, I have to show a form with the street, house, and apartment number.